### ISSUE TITLE
missing dictionaries/README-FIRST file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#163]
```
Documentation in the source code and on 
http://code.google.com/p/skipfish/wiki/SkipfishDoc says to read the 
dictionaries/README-FIRST file but it does not appear to exist in the 2.09b 
source:

$ grep -r 'README-FIRST' *
README:Next, you need to read the instructions provided in 
dictionaries/README-FIRST
README:more detail in dictionaries/README-FIRST. Please read that file before
doc/skipfish.1:Make sure you've read the instructions provided in 
dictionaries/README-FIRST to select the right dictionary file and configure it 
correctly. This step has a profound impact on the quality of scan results later 
on.
src/database.c:      PFATAL("Unable to open read-write wordlist '%s' (see 
dictionaries/README-FIRST).", fname);

$ ls -1 dictionaries/
complete.wl
extensions-only.wl
medium.wl
minimal.wl

$ tar -tvf ../skipfish-2.09b.tar | grep README
-r--r-----  0 heinenn eng     26616 Sep  1 08:28 skipfish-2.09b/README
```

Original issue reported on code.google.com by `dimi...@gmail.com` on 18 Sep 2012 at 12:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `niels.he...@gmail.com` on 18 Sep 2012 at 3:35
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `niels.he...@gmail.com` on 18 Sep 2012 at 3:35
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Looks like the dictionary/README-FIRST file was moved to doc/dictionaries.txt 
between 2.07b and 2.08b.
```

Original comment by `gene.got...@coveros.com` on 17 Nov 2012 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting ! This is fixed in 2.10b which is now in SVN .
```

Original comment by `niels.he...@gmail.com` on 23 Nov 2012 at 8:56
- Changed state: **Fixed**
